db/fixtures/miq_product_features.yml

```diff
@@ -976,6 +976,11 @@
       :description: Request to Provision Hosts / Nodes
       :feature_type: admin
       :identifier: host_miq_request_new
+    - :name: Register Hosts / Nodes
+      :description: Register new Hosts / Nodes to Ironic
```

Comment:
I think we probably want to remove this one last public-facing reference to Ironic